bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/ShellyHandlerFactory.java

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.shelly.internal;
+
+import static org.openhab.binding.shelly.internal.ShellyBindingConstants.*;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.CopyOnWriteArraySet;
+
+import org.apache.commons.lang.Validate;
```

Comment:
We are just trying to get rid off this dependency (see https://github.com/openhab/openhab-core/pulls), so it would be good if you could just to normal null checks or similar in your code.

Reply:
ok, then I do it after the release
couldn't we integrate something similar - if null checks require more lines and look nasty if you have multiple 

I would like to push this out after the release with the given timeline - is that ok for you?

Reply:
Yes, removing this dependency can be done after the release - but it would be nice if you could take care of it then, so that it won't be forgotten.